keps/sig-node/688-pod-overhead/kep.yaml

```diff
@@ -0,0 +1,20 @@
+title: Pod Overhead
+kep-number: 688
+authors:
+  - "@egernst"
+owning-sig: sig-node
+participating-sigs:
+  - sig-scheduling
+  - sig-autoscaling
+  - sig-windows
+reviewers:
+  - "@tallclair"
+  - "@derekwaynecarr"
+  - "@dchen1107"
+approvers:
+  - "@dchen1107"
+  - "@derekwaynecarr"
+editor: Eric Ernst
+creation-date: 2019-02-26
+last-updated: 2020-01-29
+status: implemented (alpha)
```

Comment:
kinda surprised that passes validation

Reply:
We're not checking for exact match. But whether there is a substring that matches the pattern :(